core/client/fs/src/main/java/alluxio/client/file/FileSystemMasterClient.java

```diff
@@ -78,7 +76,7 @@ public static FileSystemMasterClient create(MasterClientContext conf) {
    *
    * @param path the directory path
    * @param options method options
-   * @throws AlreadyExistsException if the directory already exists
+   * @throws AlluxioStatusException if the directory already exists
```

Comment:
`AlluxioStatusException` is a general type of exception like `IOException`. For the specific case of directory already existing, we throw `AlreadyExistsException`, which is a subclass of `AlluxioStatusException`.

There is a similar `NotFoundException` that also extends `AlluxioStatusException`

Can we use the specific exception type to be more accurate?